provider/azureProvider.js

```diff
@@ -227,64 +199,58 @@ class AzureProvider {
 
   getAdminKey () {
     const options = {
-      'host': functionAppName + constants.scmDomain,
-      'port': 443,
-      'path': constants.masterKeyApiApth,
-      'headers': {
-        'Authorization': constants.bearer + principalCredentials.tokenCache._entries[0].accessToken,
-        'Content-Type': constants.jsonContentType
+      url: `https://${functionAppName}${config.scmDomain}${config.masterKeyApiPath}`,
+      json: true,
+      headers: {
+        Authorization: config.bearer + principalCredentials.tokenCache._entries[0].accessToken
```

Comment:
Yes! I really really really considered doing that in this set. It's on the list and thank you for bringing it up. 👍